packages/currency/src/index.js

```diff
@@ -18,14 +19,14 @@ import { numberFormat } from '@woocommerce/number';
  * @returns {?String} A formatted string.
  */
 export function formatCurrency( number, currencySymbol ) {
-	// default to wcSettings (and then to $) if currency symbol is not passed in
+	// default to wc-admin-settings (and then to $) if currency symbol is not passed in
```

Comment:
I think `(and then to $)` should be removed from the comment, now. I would even say this comment could be completely removed, it doesn't seem to add any valuable information that can't be inferred from the code.

Reply:
Ya good call, I mostly just went through and replaced old strings and missed the context.